### ISSUE TITLE
Android TV support possible with Oreo?

### ISSUE THREAD

**Issue by bigoulours** [abertschi/ad-free#35]
Hi,
great app! I wonder if ad-free would work on Android TV Oreo since it displays notifications while playing music.

**Comment by abertschi** [comment]
As far as i know there are no official APIs to inspect notification content on Android TV. But i could be wrong. Can you upload a picture of an advertisement notification on Android TV as well as a normal playback notification?

**Comment by bigoulours** [comment]
Hi,
i managed to take screenshots of the notifications (music playing, ad and Spotify announcement).
Looks similar to phone notifications but i don't know if these are accessible through API...
![Screenshot_20190725-194817-01](https://user-images.githubusercontent.com/29359842/61897636-c5412400-af17-11e9-95b4-506fb85c1046.jpeg)
![Screenshot_20190725-194720-01](https://user-images.githubusercontent.com/29359842/61897662-d4c06d00-af17-11e9-8117-2ce0377a9b11.jpeg)
![Screenshot_20190725-191359-01](https://user-images.githubusercontent.com/29359842/61897676-dd18a800-af17-11e9-8ae4-4c7fd4d65b9f.jpeg)

**Comment by 2803media** [comment]
For Android auto there is an api https://developer.android.com/training/cars/notifications

**Comment by abertschi** [comment]
Yes, notifications and notification listeners seem to be available, on Car and TV. [This post](https://stackoverflow.com/questions/49070886/notificationlistenerservice-in-androidtv) claims that only system apps can attach themselves as notification listeners. Can you check if in the Car/ or TV settings there is an option to grant permissions for a notification listener -- similar to the phone situation? ("Settings/ Special App access/ Notification access") If so there is a chance to get this working.

**Comment by 2803media** [comment]
If you install the android auto app you will see that there is an option to display notification from medias (music, podcast or other) but the menu is different from the phone settings

**Comment by 2803media** [comment]
https://www.androidpolice.com/2019/10/04/android-auto-v4-7-app-filtering-launcher-media-notifications-optional-apk-teardown/
